squid/.gitignore

```diff
@@ -1,3 +0,0 @@
-node_modules
```

Comment:
Can we add this back?

Reply:
I moved it to the root level as it's not necessary to duplicate the same configuration. Also, it's a good practice to have configuration files like eslintrc, prettierrc, nvmrc, gitignore on the root level to avoid configuring the same things in one project.

Reply:
Agreed, there are many such things not really clean in the repo, thanks for pointing that out.

we should just eventually do this in its own PR, and make sure we don't break much.